src/components/ProjectSearch/TextSearch.js

```diff
@@ -153,28 +160,23 @@ export default class TextSearch extends Component {
       }
     });
 
-    return (
-      <span className="line-value">
-        {matches}
-      </span>
-    );
+    return dom.span({ className: "line-value" }, ...matches);
```

Comment:
I think we should be able to spread, but it doesn't work for me...

https://github.com/Microsoft/TypeScript/pull/11591/files#diff-36d63f88d66eddff8e2d469bea89b21cR23